bench/bench_test.go

```diff
@@ -159,29 +160,29 @@ func benchmarkGetMetadatas(b *testing.B, n, hvkLen, mdLen int) {
 	b.StopTimer()
 }
 
-func getMultihashes(b *testing.B, mhs []multihash.Multihash, store *dhstore.PebbleDHStore) {
+func getMultihashes(b *testing.B, mhs []multihash.Multihash, store dhstore.DHStore) {
 	for _, mh := range mhs {
 		evks, err := store.Lookup(mh)
 		require.NoError(b, err)
 		require.NotNil(b, evks)
 	}
 }
 
-func putMultihashes(b *testing.B, mhs []multihash.Multihash, vks [][]byte, store *dhstore.PebbleDHStore) {
+func putMultihashes(b *testing.B, mhs []multihash.Multihash, vks [][]byte, store dhstore.DHStore) {
```

Comment:
What is the reason for not passing in a `DHStore`  interface?

Reply:
not sure i follow; `dhstore.DHStore` is the interface.

Reply:
Nevermind, must have read the diff backward.